app.py

```diff
@@ -166,6 +168,17 @@ def get_authorization_header():
     }
 
 
+def transform_api_data(details):
```

Comment:
If this function is only used inside this file, could we prefix it with an underscore: `_tranform_api_data`?